makefile

```diff
@@ -26,6 +31,15 @@ operator-chart:
 
 deploy: .id deploy/cluster.yaml generate operator-chart operator-chart operator-todo
 
+image: .image-$(GIT_COMMIT)
+
+push: image
+	docker push $(REGISTRY)/$(IMAGE_REPO):$(GIT_COMMIT)
+
+.image-$(GIT_COMMIT):
+	OPERATOR_NAME=$(OPERATOR_NAME) .bin/operator-sdk-$(OPERATOR_SDK_VERSION) build $(REGISTRY)/$(IMAGE_REPO):$(GIT_COMMIT) --verbose
```

Comment:
Will this pack all that we need into the image? Using the kops binary approach, I have to install AWS CLI and the kops binary onto the container for it to work. I am not sure if this approach needs a Dockerfile or something to go along with it that will add what we need.

Reply:
No it's just the first step of creating the binary. I'll make more PRs for the other steps of container creation process

Reply:
👍 I have been working on the docker file, chart and binary approach in this branch https://github.com/infobloxopen/cluster-operator/tree/container

Reply:
Will you open a draft PR for it